bin/verilator

```diff
@@ -910,6 +911,16 @@ backtrace on exit, then exit GDB immediately.  Without --debug or if GDB
 doesn't seem to work, this flag is ignored.  Intended for easy creation of
 backtraces by users; otherwise see the --gdb flag.
 
+=item --generate-waivefile <filename>
```

Comment:
Suggest --waiver-output

"waive" sounds too much like "waves" which is something different entirely ;)
We already have --xml-output so seems a similar name would be good.